[2024-03-18T08:41:36.346+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sftp_sync.sync_files_task manual__2024-03-18T08:41:30.174064+00:00 [queued]>
[2024-03-18T08:41:36.414+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sftp_sync.sync_files_task manual__2024-03-18T08:41:30.174064+00:00 [queued]>
[2024-03-18T08:41:36.417+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-18T08:41:36.450+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): sync_files_task> on 2024-03-18 08:41:30.174064+00:00
[2024-03-18T08:41:36.484+0000] {standard_task_runner.py:60} INFO - Started process 1969 to run task
[2024-03-18T08:41:36.509+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sftp_sync', 'sync_files_task', 'manual__2024-03-18T08:41:30.174064+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/sftp_sync_between_source_and_target.py', '--cfg-path', '/tmp/tmpusl2k4a3']
[2024-03-18T08:41:36.517+0000] {standard_task_runner.py:88} INFO - Job 90: Subtask sync_files_task
[2024-03-18T08:41:36.661+0000] {task_command.py:423} INFO - Running <TaskInstance: sftp_sync.sync_files_task manual__2024-03-18T08:41:30.174064+00:00 [running]> on host fa05d1b87173
[2024-03-18T08:41:36.813+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sftp_sync' AIRFLOW_CTX_TASK_ID='sync_files_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-18T08:41:30.174064+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-18T08:41:30.174064+00:00'
[2024-03-18T08:41:36.819+0000] {ssh.py:301} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-03-18T08:41:36.843+0000] {transport.py:1909} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-03-18T08:41:37.077+0000] {transport.py:1909} INFO - Authentication (password) successful!
[2024-03-18T08:41:37.182+0000] {sftp.py:169} INFO - [chan 0] Opened sftp connection (server version 3)
[2024-03-18T08:41:37.190+0000] {ssh.py:301} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-03-18T08:41:37.225+0000] {transport.py:1909} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-03-18T08:41:37.586+0000] {transport.py:1909} INFO - Authentication (password) successful!
[2024-03-18T08:41:37.682+0000] {sftp.py:169} INFO - [chan 0] Opened sftp connection (server version 3)
[2024-03-18T08:41:37.697+0000] {logging_mixin.py:188} INFO - Transforming files...
[2024-03-18T08:41:37.702+0000] {sftp_sync_between_source_and_target.py:80} INFO - Syncing file file_2.txt...
[2024-03-18T08:41:37.706+0000] {python.py:202} INFO - Done. Returned value was: Synced 1 new files.
[2024-03-18T08:41:37.838+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=sftp_sync, task_id=sync_files_task, execution_date=20240318T084130, start_date=20240318T084136, end_date=20240318T084137
[2024-03-18T08:41:37.938+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-18T08:41:38.068+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
